TCP_Server.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, greetings, host, port):
         # self.main_socket.setblocking(False)
         self.greetings = greetings
         self.closed = False
+        self.EOT = '\n\x04\n'
```

Comment:
This EOT is technical sign. 
Can you move it to connection.send method ?
I think that this EOT should be hide in the communication interface/class.
In the current approach every class that want too communicate must know about this string.